bokehjs/src/coffee/range/data_range1d.coffee

```diff
@@ -35,8 +32,13 @@ class DataRange1d extends Range1d.Model
       '_end', 'flipped', '_auto_end', 'range_padding', 'default_span'
     ])
 
-    @register_property('auto_ranging', @_get_auto_ranging, true)
-    @add_dependencies('start', this, ['_start', '_end'])
+    if attrs?.start?
+      @set('start', attrs.start)
+      attrs.start = undefined
```

Comment:
why you set `undefined` here? I guess it should be to detect new attrs in the next call...